modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc

```diff
@@ -0,0 +1,45 @@
+// Ths module is included in the following assembly:
+//
+// *openshift_pipelines/working-with-pipelines-using-developer-perspective.adoc
+
+[id="interacting-with-pipelines-using-the-developer-perspective_{context}"]
+= Interacting with Pipelines using the Developer perspective
+
+The *Pipelines* view in the *Developer* perspective lists all the pipelines in a project along with details, such as the namespace in which the pipeline was created, the last pipeline run, the status of the tasks in the pipeline run, the status of the pipeline run, and the time taken for the run.
+
+[Discrete]
+=== Procedure
+. In the *Pipelines* view of the *Developer* perspective, select a project from the *Project* drop-down to see the pipelines in that project.
++
+image::op-pipeline-list.png[Pipelines in the Developer perspective]
++
+. Click on the required pipeline to see the *Pipeline Details* page. It provides a visual representation of all the serial and parallel tasks in the pipeline. The tasks are also listed to the bottom-right of the page. You can click the listed *Tasks* to view the details of the task.
++
+image::op-pipeline-details.png[Pipeline details]
++
+. Optionally, in the *Pipeline Details* page :
+
+* Click *Pipeline Runs* tab to see the completed, running, or failed runs for the pipeline. You can use the Options menu (icon image)  to: stop a running pipeline, rerun a pipeline using the same parameters and resources as that of the previous pipeline execution, or delete a pipeline run.
+* Click *Parameters* tab to see the parameters defined in the Pipeline. You can also add or edit additional parameters as required.
+* Click *Resources* tab to see the resources defined in the Pipeline. You can also add or edit additional resources as required.
+
+. Click *Actions* and select *Start* to start the pipeline. The *Start Pipeline* dialog box displays the *Git Resources* and the *Image Resources* based on the pipeline definition.
+.. In the *Parameters* section, enter a unique name in the `IMAGE_NAME` field.
```

Comment:
Did you follow some `+Add` flow? This field name is quite specific and might be confusing for customers. We should decide whether it makes sense to keep this doc as general as possible or want to introduce some specific example.

Reply:
If you are taking reference from the +Add flow then this would be `APP_NAME` and mostly pre-populated

Reply:
@sanketpathak, Initially, these docs were created using Yaml files, then once add flow was in, instructions specific to that was added. See my commented out note on line 29, post our discussions. Now, the Add flow is in DP so it will not be documented. So we will need to keep only the documentation that is relevant for pipelines created using YAML or CLI.
@ppitonak for 4.4 we will need to keep this doc very general, once the add flow is TP we can add a specific example. Considering this, what do you suggest?

Reply:
@nkakkar81 please comment out this section from line 26 onward. This is part of the Add flow which is DP for now.